python/lsst/meas/extensions/psfex/psfexPsfDeterminer.py

```diff
@@ -104,9 +104,11 @@ class PsfexPsfDeterminerConfig(measAlg.BasePsfDeterminerConfig):
         dtype=bool,
         default=False,
     )
-
-    def setDefaults(self):
-        self.kernelSize = 41
+    kernelSize = pexConfig.Field(
+        doc="WHAT DOES THIS MEAN IN THIS CONTEXT?",
```

Comment:
Could we have a more useful doc here? I don't have a suggestion, but this probably shouldn't get into `master` like this.